be/src/column/binary_column.cpp

```diff
@@ -18,21 +18,23 @@
 
 namespace starrocks::vectorized {
 
-void BinaryColumn::check_or_die() const {
+template <typename T>
+void BinaryColumnBase<T>::check_or_die() const {
     CHECK_EQ(_bytes.size(), _offsets.back());
-    for (uint32_t i = 1; i < _offsets.size(); i++) {
+    for (size_t i = 1; i < _offsets.size(); i++) {
         CHECK_GE(_offsets[i], _offsets[i - 1]);
     }
```

Comment:
Can this loop clause be eliminated by compiler? Otherwise it will cause bad performance